### DISCUSSION TITLE
Expedition 6 The Blighted

### DISCUSSION THREAD

**Discussion by Alanray64** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#49] [Q&A] [answered]
Ever since I managed to use this offline, after the advice you previously provided, I have been working through the expeditions and having fun. Today though I was just two off completing the Blighted, these being go to the Galactic Centre and Establish a presence there.

My problem is I am unable to reach this Galactic Core? I select the Galactic Map, I use the left or right arrows to choose the centre and then try to get what everyone else is saying is easy. I am supposed to move the mouse around so the line lights up in the direction to the core however this just moves me to the next system. I really have no idea where this circle people talk about appearing and fills up.. For me it just pops me along the route as far as I can go. I do have hyperdrive set at 100%, I am using a Class B Sentinel Ship, but I was wondering if this is a bug or something?

**Comment by cwmonkey** [comment, marked as answer]
lol yeah I had a hard time with it the first couple times I tried. Good news is I just did this expedition on my current Permadeath run, so I'm sure it's working.

Basically you go to the last rendezvous and point your path off the screen and hold the left mouse button down. I feel like it's usually oriented up when I do it, but that could just be me spinning the camera.

Here's the first video on YouTube I found that quickly demonstrates it:

https://www.youtube.com/watch?v=0fTxppbwHyQ

**Reply by Alanray64** [reply]
Bloody brilliant. I went back to the phase 5 point, went out from the space station and did the transition. Worked like a charm. 

Thanks heaps for your thoughts 👍
